frontend-react/src/pages/support/legacy-page-content/Contact.tsx

```diff
@@ -60,6 +63,14 @@ export const Contact = () => {
                     </a>
                     .
                 </p>
+                <p>
```

Comment:
if we wanted to introduce a feature flag I guess we would put a conditional here to hide this paragraph?